Lib/test/test_zstd.py

```diff
@@ -2430,7 +2430,6 @@ def test_buffer_protocol(self):
             self.assertEqual(f.write(arr), LENGTH)
             self.assertEqual(f.tell(), LENGTH)
 
-@unittest.skip("it fails for now, see gh-133885")
 class FreeThreadingMethodTests(unittest.TestCase):
 
     @unittest.skipUnless(Py_GIL_DISABLED, 'this test can only possibly fail with GIL disabled')
```

Comment:
We should remove this `@unittest.skipUnless(Py_GIL_DISABLED, ...);`. It's very possible for the test to fail with the GIL enabled.

Reply:
Removed this, it's a good point!